kore/src/Kore/Internal/InternalMap.hs

```diff
@@ -45,26 +45,26 @@ type MapElement = Element NormalizedMap
 
 -- | Wrapper for normalized maps, to be used in the `builtinAcChild` field.
 newtype NormalizedMap key child = NormalizedMap {getNormalizedMap :: NormalizedAc NormalizedMap key child}
-    deriving (Eq, Ord, Show)
-    deriving (Foldable, Functor, Traversable)
-    deriving (GHC.Generic)
+    deriving stock (Eq, Ord, Show)
+    deriving stock (Foldable, Functor, Traversable)
+    deriving stock (GHC.Generic)
     deriving anyclass (Hashable, NFData)
     deriving anyclass (SOP.Generic, SOP.HasDatatypeInfo)
     deriving anyclass (Debug, Diff)
 
 instance AcWrapper NormalizedMap where
     newtype Value NormalizedMap child = MapValue {getMapValue :: child}
-        deriving (Eq, Ord, Show)
-        deriving (Foldable, Functor, Traversable)
-        deriving (GHC.Generic)
+        deriving stock (Eq, Ord, Show)
+        deriving stock (Foldable, Functor, Traversable)
```

Comment:
Switching to newtype deriving: the actual error seems to be the `cannot eta-reduce` one.